static/css/targets.less

```diff
@@ -0,0 +1,139 @@
+  align-items: stretch;
+  flex-flow: row wrap;
+  justify-content: center;
+  padding: 35px 0;
+
+  .target {
+    padding: 10px;
+    width: 350px;
+    min-height: 180px;
+    background-color: #FFFFFF;
+    border: 1px solid @seperator-color;
+    border-radius: @radius-size-small;
+    margin: 15px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    &.has-goal,
+    &.has-goal:not(.goal-met) .percent {
+      .number {
+        color: @high-risk-color;
+      }
+      .progress-bar {
+        background-color: @high-risk-color;
+      }
+    }
+    &.has-goal.goal-met,
+    .percent {
+      .number {
+        color: @analytics-color;
+      }
+      .progress-bar {
+        background-color: @analytics-color;
+      }
+    }
+    &:not(.has-goal) .goal {
+      display: none;
+    }
+
+    .heading {
+      .title {
+        margin-left: 70px;
+        h2 {
+          margin: 0;
+        }
+        p {
+          color: @label-color;
+        }
+      }
+      .icon {
+        float: left;
+        img {
+          width: 60px;
+        }
+      }
+    }
+    .body {
+      clear: both;
+      position: relative;
+      margin: 0 5px;
+      .percent {
+        padding-top: 10px;
+        position: relative;
+        .goal {
+          position: absolute;
+          top: 0;
+          label {
+            margin-left: -50%; /* center aligned */
+            white-space: nowrap;
+            font-size: @font-small;
+            color: @label-color;
+            padding-top: 5px;
+            &.pin-left {
+              margin-left: 0; /* left aligned */
+            }
+            &.pin-right {
+              margin-left: -100%; /* right aligned */
+            }
+          }
+          .bar {
+            position: absolute;
+            top: 25px;
+            height: 30px;
+            width: 8px;
+            margin-left: -4px; /* center */
+            background-color: @label-color;
+            border: 2px solid #FFFFFF;
+            border-radius: @radius-size;
+          }
+        }
+        .value {
+          font-size: @font-extra-extra-large;
+          line-height: 1;
+          font-weight: bold;
+        }
+      }
+      .count {
+        text-align: center;
+        .number {
+          font-size: 56px;
+          line-height: 1;
+          font-weight: bold;
+          margin-bottom: 15px;
+        }
+        .goal {
+          position: absolute;
+          right: 0;
+          bottom: 20px;
+          color: @label-color;
+          p {
+            font-size: @font-extra-extra-large;
+            line-height: 1;
+            margin-bottom: 0;
+          }
+          label {
+            font-size: @font-small;
+            line-height: 1;
+            margin-bottom: 0;
+          }
+        }
+      }
+    }
+  }
+}
+
+@media (max-width: 400px) {
+  .targets {
+    padding: 0;
+    .target {
+      width: 100%;
+      margin: 0 0 8px 0;
+      border-radius: 0;
```

Comment:
I didn't think you needed to do this by default? Or is this overriding something else?

Reply:
This is overriding the non-mobile styles.